src/Apps/Collect/Components/Filters/ColorFilter.tsx

```diff
@@ -0,0 +1,229 @@
+import { CheckIcon } from "@artsy/palette"
+import { FilterState } from "Apps/Collect/FilterState"
+import { ContextConsumer } from "Artsy/SystemContext"
+import React from "react"
+
+interface Props {
+  filters: FilterState
+}
+
+const CheckmarkPositions = {
+  red: {
+    bottom: "63px",
+    left: "137px",
+  },
+  "black-and-white": {
+    bottom: "98px",
+    left: "79px",
+  },
+  pink: {
+    bottom: "98px",
+    left: "145px",
+  },
+  darkviolet: {
+    bottom: "132px",
+    left: "138px",
+  },
+  violet: {
+    bottom: "156px",
+    left: "112px",
+  },
+  darkblue: {
+    bottom: "165px",
+    left: "79px",
+  },
+  lightblue: {
+    bottom: "156px",
+    left: "43px",
+  },
+  darkgreen: {
+    bottom: "132px",
+    left: "20px",
+  },
+  lightgreen: {
+    bottom: "98px",
+    left: "10px",
+  },
+  yellow: {
+    bottom: "63px",
+    left: "20px",
+  },
+  gold: {
+    bottom: "36px",
+    left: "43px",
+  },
+  orange: {
+    bottom: "28px",
+    left: "79px",
+  },
+  darkorange: {
+    bottom: "36px",
+    left: "112px",
+  },
+}
+
+export class ColorFilter extends React.Component<Props> {
+  toggleColor(color, filters, mediator) {
+    if (filters.state.color === color) {
+      filters.unsetFilter("color", mediator)
+    } else {
+      filters.setFilter("color", color, mediator)
+    }
+  }
+
+  render() {
+    const { filters } = this.props
+
+    return (
+      <ContextConsumer>
+        {({ mediator }) => {
+          const CheckmarkStyle = {
+            position: "relative",
+            ...CheckmarkPositions[filters.state.color],
+          }
+          return (
+            <>
```

Comment:
Can simplify all of the svg props down below a bit by turning this fragment into something like 

```js
const Container = styled.div`
  cursor: pointer;
`
```

and then remove all of the individual `style = {{ cursor: 'pointer' }}` from each node.